app/js/services/situationService.js

```diff
@@ -187,8 +187,16 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
 
         flattenPatrimoine: flattenPatrimoine,
 
+        getDemandeur: function(situation) {
+            return _.find(situation.individus, { role: 'demandeur' });
+        },
+
         hasEnfantScolarise: function(situation) {
             return !!_.find(situation.individus, { role: 'enfant', scolarite: 'college' }) || !!_.find(situation.individus, { role: 'enfant', scolarite: 'lycee' });
+        },
+
+        hasEnfant: function(situation) {
+            return !!_.find(situation.individus, { role: 'enfant' });
```

Comment:
> So in JS, as long as the code is fine with it, you shouldn't cast ?

Well, it is a matter of consistency, of course. If your codebase relies on `=== true`, you should obviously cast. On the other hand however, if you do take advantage of truthies and falsies and write `if (hasEnfant)`, what's the point of casting it? Wasting (admittedly very little) CPU and memory?

As soon as you understand that _everything_ in JS can be evaluated to a boolean value, there is little need for actual `Boolean` types, except in domain representations and specific cases where you would have a different treatment for different falsies (such as `undefined` ≠ `false`, as we've seen above).
This doesn't mean you should _avoid_ `Boolean` values. It simply means `return myVal` is as much a boolean value as `return !! myVal`. The fact that the function name suggests it returns a `true`/`false` value actually (to me) suggests it is very safe to return truthies/falsies, as its return value would be only used in a comparison or a condition.

Then again, consistency above all. But if you want to return a `Boolean`, I'd go with a clear, readable cast where even a JS newbie understands what goes on rather than the cryptic `!!`.